test/dynamics/webApi/read.test.js

```diff
@@ -671,7 +671,7 @@ describe('Dynamics - read', async () => {
 
       const expectedRequest = {
         'method': 'GET',
-        'url': `${dynamicsRoot}/defra_lobserviceuserlinks?%24filter=(%20_defra_serviceuser_value%20eq%20c20e6efe-9954-4c5b-a76c-83a5518a1385%20)%20and%20statuscode%20eq%201%20and%20(%20_defra_servicerole_value%20eq%20c20e6efe-9954-4c5b-a76c-83a5518a1385%20or%20_defra_servicerole_value%20eq%20c20e6efe-9954-4c5b-a76c-83a5518a1385%20)&%24expand=defra_ServiceRole`,
+        'url': `${dynamicsRoot}/defra_lobserviceuserlinks?%24filter=(%20_defra_serviceuser_value%20eq%20c20e6efe-9954-4c5b-a76c-83a5518a1385%20)%20and%20statuscode%20eq%201&%24expand=defra_ServiceRole`,
```

Comment:
Representation hard to read

Reply:
Agree, but that's how it comes out the other end